apps/system/style/chrome/chrome.css

```diff
@@ -243,7 +248,8 @@
 
 
 /* Private browser icon in rocketbar */
-.appWindow.private > .chrome.maximized .title {
+.appWindow.private > .chrome-plain.maximized .title,
```

Comment:
There's no padding on .chrome-combined .title at all now, its all on .chrome-title-container. If I could make it display:inline I would, but we need block for the text-overflow ellipsis to work. For this rule, I changed the selector from chrome-plain.maximized .title to chrome-plain.maximized .title - those are exclusive - chrome-combined is never chrome-plain and vice-versa.